x-pack/plugins/security_solution/public/common/store/sourcerer/helpers.ts

```diff
@@ -10,6 +10,17 @@ import { SourcererDataView, SourcererModel, SourcererScopeById, SourcererScopeNa
 import { SelectedDataViewPayload } from './actions';
 import { sourcererModel } from '../model';
 
+export const sortWithExcludesAtEnd = (indices: string[]) => {
```

Comment:
Nit: Can we add a quick unit test for the new helper?

Reply:
➕ 1